Templating/VOKTemplateModel.m

```diff
@@ -25,7 +25,7 @@ @interface VOKTemplateModel ()
 static NSString *const KitNameIOS = @"UIKit";
 static NSString *const KitNameMac = @"AppKit";
 
-static NSString *const FrameworkPrefix = @"ac_";
+static NSString *const MethodNamePrefix = @"ac";
```

Comment:
I'll have another go with it, too. I didn't care for the "solution" of hard-coding the `_` in the template. I'll let you know if I see different results.